dbflow/src/main/java/com/xmartlabs/base/dbflow/controller/DbFlowController.java

```diff
@@ -1,4 +1,4 @@
-package com.xmartlabs.template.controller;
+package com.xmartlabs.base.dbflow.controller;
```

Comment:
That's the other option, and i'm fine either or.
The reason I named that is because we could potentially have more than one db or service implementation, i.e. both the service and orm modules are not generic but rather an implementation for a concrete library.

@bryant1410 @michaelEllisUy @chacaa What do you think about it? Do you prefer orm and service or the specific dbflow and retrofit?